src/styles/blocks/page.scss

```diff
@@ -0,0 +1,7 @@
+* {
```

Comment:
imho general styles shouldn't be in `blocks` folder. you also can directly style `html` tag, there is no need in class for that, but it's not a mistake